### PR TITLE
ENH: Resized table, adjusted text alignment, and centered table in the page.

### PR DESCRIPTION
**Pull request by Davrod29**
Brief summary of the Pull Request, including any issues it may fix using the GitHub closing syntax:

https://help.github.com/articles/closing-issues-using-keywords/

Also, include any co-authors or contributors using the GitHub coauthor tag:

https://help.github.com/articles/creating-a-commit-with-multiple-authors/

---

Include any questions for reviewers, screenshots, sample outputs, etc.

### CODE REVIEW

**Comment by Davrod29** [comment]
Thank you for the feedback. I have added a new stylesheet and removed all changes made to normalize. Please see latest change to this request.

File: q2templates/templates/assets/css/base-template.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+/**
+* Center table heads in each column and related text in table.
+* Center table in page with reduced size.
+*/
+   
+div.text-center {
+  margin-left: 10%;
+  justify-content: center;
+  width: 80%;
```
Comment (ebolyen) [inline q2templates/templates/assets/css/base-template.css]: This makes me a little bit nervous, as the CSS selector looks relatively generic. Could you elaborate on the purpose of the percentage values (and where the `text-center` class generally appears)?

File: q2templates/templates/assets/css/base-template.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/*! normalize.css v5.0.0 */
```
Comment (ebolyen) [inline q2templates/templates/assets/css/base-template.css]: You can drop this bit :)

File: q2templates/templates/assets/css/base-template.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+/**
+ * 1. Change the width and margin values for table populated in base.html.
+ * 2. Center the data cells and table headers.
+ */
+
+/**
+* Center table heads in each column and related text in table.
+* Center table in page with reduced size.
+*/
```
Comment (ebolyen) [inline q2templates/templates/assets/css/base-template.css]: Can you drop these comments?

File: q2templates/templates/assets/css/base-template.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  
+table.dataframe thead th, table.dataframe td{
+  text-align: center;
+}
```
Comment (ebolyen) [inline q2templates/templates/assets/css/base-template.css]: Really very minor, but having every file end with a newline makes unix tools like `cat` behave nicely.

**Pull request merged**